ldpd/ldp_debug.c

```diff
@@ -29,6 +29,9 @@
 struct ldp_debug conf_ldp_debug;
 struct ldp_debug ldp_debug;
 
+/* be safe since it is already defined on Apple system targets */
+#undef MSG_SEND
```

Comment:
@rwestphal -> Should we fix ldp to approach it's macro expansion slightly differently instead of doingthis?

Reply:
Yes, using token concatenation (`##`) in the DEBUG macros was probably not a good idea in the first place (this code was mostly stolen from bgp_debug.h). But I wouldn't bother with this workaround for now, this is something we can address later.

Reply:
I just spoke with @rwestphal about this issue and we believe the correct thing to do here is: remove the special macro:
```#define LDP_DEBUG(a, b)   (ldp_debug.a & LDP_DEBUG_ ## b)```

and replace it with:
```#define LDP_DEBUG(a, b)  (ldp_debug.a & b)```

```if (LDP_DEBUG(msg, LDP_DEBUG_MSG_SEND)) ``` where we call LDP_DEBUG